web/src/lib/column-filter-utils.ts

```
@@ -373,3 +385,148 @@ export function getOperations(columnType: ColumnType) {
+    }
+  }
+
+  if (columnId === "published") {
+    const dateValue = new Date(itemValue as string).getTime()
+    const compareDate1 = new Date(value).getTime()
+
+    if (isNaN(compareDate1)) return true
+
+    switch (operation) {
+      case "eq":
+        const d1 = new Date(dateValue)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)